pkg/backup/backup.go

```diff
@@ -116,6 +116,8 @@ func (c *Controller) Backup() error {
 					eventer.EventReasonFailedSetup,
 					err.Error(),
 				)
+			} else {
+				log.Errorf("Failed to write event on %s %s. Reason: %s\n ", restic.Kind, restic.Name, rerr)
```

Comment:
No `\n` needed. This will be added by log library automatically.
https://github.com/golang/glog/blob/master/glog.go#L643